Sources/Sentry/SentrySwizzle.m

```diff
@@ -21,7 +21,11 @@ - (SentrySwizzleOriginalIMP)getOriginalImplementation
 {
     NSAssert(_impProviderBlock, nil);
 
-    self.originalCalled = true;
+#if TEST
```

Comment:
This change was needed because Alamofire has Thread Sanitizer enabled when running tests. The Thread Sanitizer detected a race condition here, which led to hanging tests in Alamofire. I want to enable Thread Sanitizer for our tests in a future PR.